mitmproxy/proxy/layers/http/_http_h2.py

```diff
@@ -49,6 +49,8 @@ class BufferedH2Connection(h2.connection.H2Connection):
 
     def __init__(self, config: h2.config.H2Configuration):
         super().__init__(config)
+        self.local_settings.initial_window_size = 2**24
+        self.local_settings.acknowledge()
```

Comment:
Let's copy over the comment from https://github.com/mitmproxy/mitmproxy/blob/00e033e8aec0cc85ca5d029cd127be77fe2f48f9/mitmproxy/proxy/layers/http/_http2.py#L466, this is a stupid API pitfall we should keep pointing out. :)